### PR TITLE
describe session log

### PR DESCRIPTION
**Pull request by itayd**
not the most beautiful thing, but easier to read:

```
$ ./bin/ak session log
[2024-07-10T00:32:26Z] State: CREATED
[2024-07-10T00:32:26Z] State: RUNNING
  RunID: run_01j2d0wjvdf4eaba234r995b7f

[2024-07-10T00:32:26Z] State: RUNNING on_http_post
  Call:
    on_http_post(data)
    Executor: run_01j2d0wjvdf4eaba234r995b7f
    Data: 14 bytes
  RunID: run_01j2d0wjvdf4eaba234r995b7f

[2024-07-10T00:32:26Z] State: ERROR
  Error:
    Extra:
      cause: floating-point division by zero
      raw: floating-point division by zero
      type: eval
    Value:
      "floating-point division by zero"
    Callstack:
      [0] http.star:30.12,on_http_post

```

```
[2024-07-09T23:44:57Z] State: COMPLETED 
  Prints:
    2024-07-09 23:44:56.730481 +0000 UTC
    OK
    http_response(body = body(bytes = , json = , text = ), encoding = "", headers = {"Date": "Tue, 09 Jul 2024 23:44:57 GMT", "Access-Control-Allow-Origin": "*", "Access-Control-Allow-Credentials": "true", "Server": "gunicorn/19.9.0", "Content-Type": "text/html; charset=utf-8", "Content-Length": "0"}, status_code = 404, url = "https://httpbin.org/status/404") None
    
  Exports:
    on_http_test: "|function: on_http_test|"
    foo: "|function: foo|"
    on_http_get: "|function: on_http_get|"
    on_http_post: "|function: on_http_post|"
  Return Value:
    null

[2024-07-09T23:44:57Z] Print: 

[2024-07-09T23:44:57Z] Print: http_response(body = body(bytes = , json = , text = ), encoding = "", headers = {"Date": "Tue, 09 Jul 2024 23:44:57 GMT", "Access-Control-Allow-Origin": "*", "Access-Control-Allow-Credentials": "true", "Server": "gunicorn/19.9.0", "Content-Type": "text/html; charset=utf-8", "Content-Length": "0"}, status_code = 404, url = "https://httpbin.org/status/404") None

[2024-07-09T23:44:57Z] Call Attempt Complete
  Final
  Return Value: 
    json: json: unsupported type: map[interface {}]interface {}

[2024-07-09T23:44:56Z] Call Attempt Start: Attempt #0

[2024-07-09T23:44:56Z] Call Spec: seq #4, get
  Function:
    get(url,params?,headers?,data?,json?)
    Executor: int_3kth00httpf1201a7ed83f7cd5
    Data: 30 bytes
  Arg #0:
    "https://httpbin.org/status/404"
  Arg #1:
    {
      "__ctor": "callopts",
      "catch": true
    }

[2024-07-09T23:44:56Z] Call Attempt Complete
  Final
  Return Value: 
    {
      "__ctor": "callopts",
      "catch": true
    }

[2024-07-09T23:44:56Z] Call Attempt Start: Attempt #0

[2024-07-09T23:44:56Z] Call Spec: seq #3, callopts
  Function:
    callopts()
    Executor: int_3kth0000ak08326707b4eb37b1
    Flags: pure,no-poll
  Arg "catch":   true

[2024-07-09T23:44:56Z] Print: OK

[2024-07-09T23:44:56Z] Call Attempt Complete
  Final
  Return Value: 
    "OK"

[2024-07-09T23:44:56Z] Call Attempt Start: Attempt #0

[2024-07-09T23:44:56Z] Call Spec: seq #2, set
  Function:
    set(key,value,ttl?)
    Executor: int_3kth0st0re5a079a2f9797da68
  Arg #0:
    "x"
  Arg #1:
    "meow"

[2024-07-09T23:44:56Z] Print: 2024-07-09 23:44:56.730481 +0000 UTC

[2024-07-09T23:44:56Z] Call Attempt Complete
  Final
  Return Value: 
    "2024-07-09T23:44:56.730481Z"

[2024-07-09T23:44:56Z] Call Attempt Start: Attempt #0

[2024-07-09T23:44:56Z] Call Spec: seq #1, now
  Function:
    now()
    Executor: int_3kth00t1me8c16217e9fd7161a
    Flags: pure,privilidged

[2024-07-09T23:44:56Z] State: RUNNING on_http_get
  Call:
    on_http_get()
    Executor: run_01j2cy5kwneb78wctgwsz0re3z
    Data: 13 bytes
  RunID: run_01j2cy5kwneb78wctgwsz0re3z

[2024-07-09T23:44:56Z] State: RUNNING 
  RunID: run_01j2cy5kwneb78wctgwsz0re3z

[2024-07-09T23:44:56Z] State: CREATED 
meow_63690_...
```

### CODE REVIEW

File: sdk/sdktypes/session_log_record_describe.go [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+		}
+		if m.CallAttemptComplete.Result.Error != nil {
+			fmt.Fprintln(w, "Error:")
+			describeError(opts, opts.indent(w), kittehs.Must1(ProgramErrorFromProto(m.CallAttemptComplete.Result.Error)))
+		} else {
+			fmt.Fprintln(w, "Return Value: ")
+			describeValue(opts, opts.indent(w), kittehs.Must1(ValueFromProto(m.CallAttemptComplete.Result.Value)))
+		}
+	default:
+		fmt.Fprintln(b, "???")
```
Comment (daabr) [inline sdk/sdktypes/session_log_record_describe.go]: Please use an informative error message that users can understand

File: sdk/sdktypes/session_log_record_describe.go [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+			describeFunctionValue(opts, opts.indent(w), call)
+		} else {
+			fmt.Fprintln(w, "")
+		}
+		fmt.Fprintf(w, "%sRunID: %s\n", opts.Indent, s.read().RunId)
+	case SessionStateCompleted:
+		fmt.Fprintln(w, "")
+		describeCompletedState(opts, opts.indent(w), s)
+	default:
+		fmt.Fprintln(w, "???")
```
Comment (daabr) [inline sdk/sdktypes/session_log_record_describe.go]: Please use an informative error message that users can understand

**Review by daabr — APPROVED** [review]

File: sdk/sdktypes/session_log_record_describe.go (hunk 1 above)
Comment (itayd) [reply sdk/sdktypes/session_log_record_describe.go]: done

File: sdk/sdktypes/session_log_record_describe.go (hunk 2 above)
Comment (itayd) [reply sdk/sdktypes/session_log_record_describe.go]: done

**Pull request merged**